example/package.json

```diff
@@ -20,7 +20,10 @@
     "react-native-app-auth": "^6.4.3",
     "react-native-config": "^1.5.0",
     "react-native-device-info": "^10.3.0",
+    "react-native-gesture-handler": "^2.9.0",
     "react-native-keychain": "^8.1.1",
+    "react-native-reanimated": "^3.0.2",
+    "react-native-svg": "^13.8.0",
```

Comment:
These three required being added to the example app as well because they have native parts requiring pod-install.  Because of this I added them also to the peer-dependencies of the sdk

Reply:
💯